source/installation/with/yunohost.html.md

```diff
@@ -2,8 +2,16 @@
 title: Installing with YunoHost
 ---
 
-This installation method is not yet documented. Your help would be welcome!
+You need to have a Yunohost server.
 
-TODO: check this link is working
+If you don't have YunoHost, please see [here](https://yunohost.org/#/install) to know how to install and enjoy it.
 
-<a href="https://contribute.joinplu.me/#/documentation">Help writing this page</a>
\ No newline at end of file
+You can install it from the [Yunohost Apps](https://yunohost.org/#/apps_en), select All apps in the dropdown menu, Search for Plume, click on install and follow the steps.
+
+or
+
+Using from an SSH connexion to your Yunohost server: 
```

Comment:
many people might be very confused with the rather rare spelling of "connexion"

please use "connection"

Reply:
My bad, usual french mistake